openforcefield/topology/topology.py

```diff
@@ -1587,10 +1591,12 @@ def to_mdtraj(self):
         mdtraj_topology : mdtraj.Topology
             An MDTraj Topology object
         """
+        import mdtraj as md
         return md.Topology.from_openmm(self.to_openmm())
 
+    # TODO: Jeff prepended an underscore on this before 0.2.0 release to remove it from the API. Fix?
     @staticmethod
-    def from_parmed(parmed_structure, unique_molecules=None):
+    def _from_parmed(parmed_structure, unique_molecules=None):
```

Comment:
Why?

Reply:
I think this one was probably unnecessary. I've removed it and added 
```
        .. warning:: This functionality will be implemented in a future toolkit release.
```
to the docstring.